docs/tutorial/part-two/index.md

```diff
@@ -103,67 +103,67 @@ Your project's file structure should now look like this:
+> Tip: Bagian ini berfokus pada cara tercepat dan termudah untuk memulai styling pada situs Gatsby — impor standar berkas CSS secara langsung menggunakan `gatsby-browser.js`. Pada kebanyakan kasus, cara terbaik menambahkan global styles adalah dengan _shared layout component_. [Telusur dokumentasi](/docs/global-css/) untuk penjelasan lebih tentang pendekatan itu.
 
-## Using component-scoped CSS
+## Menggunakan component-scoped CSS
 
-So far, we've talked about the more traditional approach of using standard css stylesheets. Now, we'll talk about various methods of modularizing CSS to tackle styling in a component-oriented way.
+Sejauh ini, kita telah membahas tentang pendekatan yang lebih tradisional tentang penggunaan standar css stylesheet. Sekarang, kita akan membahas beragam metode tentang modularisasi CSS untuk menangani styling berbasis komponen.
 
-### CSS Modules
+### Modul CSS
 
-Let's explore **CSS Modules**. Quoting from
+Mari kita jelajahi **CSS Module**. Dikutip dari
 [the CSS Module homepage](https://github.com/css-modules/css-modules):
 
-> A **CSS Module** is a CSS file in which all class names and animation names
-> are scoped locally by default.
+> **CSS Module** merupakan berkas CSS yang mana semua nama kelas dan nama animasi
+> normalnya dicakup secara lokal.
 
-CSS Modules are very popular because they let you write CSS normally but with a lot more safety. The tool automatically generates unique class and animation names, so you don't have to worry about selector name collisions.
+CSS Modules sangat populer karena CSS Modules mengizinkan Anda menulis CSS secara normal namun dengan banyak keamanan. CSS Modules secara otomatis menghasilkan nama kelas dan animasi unik, jadi Anda tidak perlu khawatir tentang penamaan _selector_ yang sama.
 
-Gatsby works out of the box with CSS Modules. This approach is highly recommended for those new to building with Gatsby (and React in general).
+Gatsby berfungsi _out of the box_ dengan CSS Modules. Pendekatan ini sangat direkomendasikan untuk mereka yang awam dalam pengembangan menggunakan Gatsby (dan React pada umumnya).
 
 #### ✋ Build a new page using CSS Modules
 
-In this section, you'll create a new page component and style that page component using a CSS Module.
+Pada bagian ini, Anda akan membuat komponen halaman baru dan style yang digunakan halaman komponen menggunakan CSS Module.
 
-First, create a new `Container` component.
+Pertama, buat komponen `Container` baru.
 
-1. Create a new directory at `src/components` and then, in this new directory, create a file named `container.js` and paste the following:
+1. Buat direktori baru pada `src/components` lalu, dalam direktori tersebut, buat berkas dengan nama `container.js` dan tempel kode berikut:
 
 ```javascript:title=src/components/container.js
-import React from "react"
-import containerStyles from "./container.module.css"
+import React from 'react';
```

Comment:
Please leave the code, dont change any of it

Reply:
sorry this is my linter again